src/elements/atoms/Flex/Flex.example.jsx

```diff
@@ -0,0 +1,409 @@
+          <Card style={{ width: '25%' }} id="flex-06">
+            <Card__header>Item Two</Card__header>
+            <Card__body><p>{Utils.ipsum('word', 40)}</p></Card__body>
+          </Card>
+          <Card style={{ width: '25%' }} id="flex-07">
+            <Card__header>Item Three</Card__header>
+            <Card__body><p>{Utils.ipsum('word', 40)}</p></Card__body>
+          </Card>
+        </FlexRow>
+      ),
+      notes: ''
+    }, {
+      name: 'Flex row, justify content/items end',
+      description: '',
+      component: (
+        <FlexRow justify="end">
+          <Card style={{ width: '25%' }} id="flex-08">
+            <Card__header>Item One</Card__header>
+            <Card__body><p>{Utils.ipsum('word', 40)}</p></Card__body>
+          </Card>
+          <Card style={{ width: '25%' }} id="flex-09">
+            <Card__header>Item Two</Card__header>
+            <Card__body><p>{Utils.ipsum('word', 40)}</p></Card__body>
+          </Card>
+          <Card style={{ width: '25%' }} id="flex-10">
+            <Card__header>Item Three</Card__header>
+            <Card__body><p>{Utils.ipsum('word', 40)}</p></Card__body>
+          </Card>
+        </FlexRow>
+      ),
+      notes: ''
+    }, {
+      name: 'Flex row, justify content/items between',
+      description: '',
+      component: (
+        <FlexRow justify="between">
+          <Card style={{ width: '30%' }} id="flex-11">
+            <Card__header>Item One</Card__header>
+            <Card__body><p>{Utils.ipsum('word', 40)}</p></Card__body>
+          </Card>
+          <Card style={{ width: '30%' }} id="flex-12">
+            <Card__header>Item Two</Card__header>
+            <Card__body><p>{Utils.ipsum('word', 40)}</p></Card__body>
+          </Card>
+          <Card style={{ width: '30%' }} id="flex-13">
+            <Card__header>Item Three</Card__header>
+            <Card__body><p>{Utils.ipsum('word', 40)}</p></Card__body>
+          </Card>
+        </FlexRow>
+      ),
+      notes: ''
+    }, {
+      name: 'Flex row, justify content/items around',
+      description: '',
+      component: (
+        <div className="flex flex--justify-content-around">
+          <Card style={{ width: '30%' }} id="flex-14">
+            <Card__header>Item One</Card__header>
+            <Card__body><p>{Utils.ipsum('word', 40)}</p></Card__body>
+          </Card>
+          <Card style={{ width: '30%' }} id="flex-15">
+            <Card__header>Item Two</Card__header>
+            <Card__body><p>{Utils.ipsum('word', 40)}</p></Card__body>
+          </Card>
+          <Card style={{ width: '30%' }} id="flex-16">
+            <Card__header>Item Three</Card__header>
+            <Card__body><p>{Utils.ipsum('word', 40)}</p></Card__body>
+          </Card>
+        </div>
+      ),
+      notes: ''
+    }, {
+      name: 'Flex row, justify content/items evenly',
+      description: '',
+      component: (
+        <div className="flex flex--justify-content-even">
```

Comment:
Same here, I think just need to update `<FlexRow>...</FlexRow>` instead if `<div>...</div>`